infra/src/dsl/xroad.ts

```diff
@@ -26,20 +26,22 @@ export type XRoadEnvs<
   T extends XroadSectionConfig
 > = T extends XroadSectionConfig ? keyof T['env'] : never
 
+export const XROAD_BASE_PATH = {
+  dev: ref((h) => h.svc('http://securityserver.dev01.devland.is')),
+  staging: ref((h) => h.svc('http://securityserver.staging01.devland.is')),
+  prod: 'http://securityserver.island.is',
+  local: ref((h) => h.svc('http://localhost:8081')), // x-road proxy
+}
 export const Base = new XroadConf({
   env: {
-    XROAD_BASE_PATH: {
-      dev: ref((h) => h.svc('http://securityserver.dev01.devland.is')),
-      staging: 'http://securityserver.staging01.devland.is',
-      prod: 'http://securityserver.island.is',
-      local: ref((h) => h.svc('http://localhost:8081')),
-    },
+    XROAD_BASE_PATH: XROAD_BASE_PATH,
```

Comment:
Why did you move the config?

Reply:
Great point, I was moving along so that was the easiest thing to do. Made an improvement.